src/components/content-types/TextContent.tsx

```diff
@@ -45,6 +52,8 @@ export default function TextContent(props: Props) {
   const [cursorPos, setCursorPos] = useState<IQuillRange | undefined>(undefined)
   const selfId = useSelfId()
 
+  useAutoAdvanceLastSeenHeads(createDocumentLink("text", props.documentId))
```

Comment:
Would it be better to make this behaviour default at the <Content/> object? Don't want to block on this but as a general principle I try to make sure <Content/> doesn't have to think about other services unless it wants to customize behaviour.